src/containers/__tests__/__snapshots__/NodeBucket.test.js.snap

```diff
@@ -1,6 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`<NodeBucket /> renders ok 1`] = `
+exports[`<NodeBucket /> does not display bucket when positioning is disabled 1`] = `
```

Comment:
Just personal preference, but looking at this snapshot (compared the other), it doesn't immediately jump out that there's no inner content rendered.

An assertion like `expect(component.children()).toHaveLength(0)` might be more clear.

Reply:
That works for me. I've updated here.